types/stake.go

```diff
@@ -1,18 +1,30 @@
 package types
 
 import (
+	"math/big"
+
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/crypto"
 )
 
 // status of a validator
 type BondStatus byte
 
-// nolint
+// staking constants
 const (
```

Comment:
The more go-y way to do this would be with `Iota` https://github.com/golang/go/wiki/Iota#examples

```go
const (
	// One different byte statuses
	One BondStatus = iota
	Two
	Three
	Four
)
```

Reply:
hmm while I do agree - I think it's also kinda nice to _see_ the different bytes (nice for reviewing the spec) I'm going to leave this as is for now but - but maybe worth switching